app/controllers/admin/pages_controller.rb

```diff
@@ -89,4 +89,12 @@ def initialize_meta_rows_and_buttons
       @meta << {:field => "slug", :type => "text_field", :args => [{:class => 'textbox', :maxlength => 100}]}
       @meta << {:field => "breadcrumb", :type => "text_field", :args => [{:class => 'textbox', :maxlength => 160}]}
     end
+
+  def verify_page_class(page_class)
+    if page_class.constantize.class == Page
```

Comment:
Yup! I tested it locally on Convoy. It is really doing the verification twice now to keep away harmful params of trying to become the User class.